internal/pkg/manifest/pipeline.go

```diff
@@ -205,6 +207,15 @@ type Deployment struct {
 	DependsOn      []string `yaml:"depends_on"`
 }
 
+// PrePostDeployments represent a directed graph of cloudformation deployments.
+type PrePostDeployments map[string]*PrePostDeployment
+
+// PrePostDeployment is the config for a pre- or post-deployment action.
+type PrePostDeployment struct {
+	BuildspecPath string   `yaml:"buildspec_path"`
```

Comment:
what do you think of just "buildspec"? Just like the current [`buildspec`](https://aws.github.io/copilot-cli/docs/manifest/pipeline/#build-buildspec) field for the build stage.
```suggestion
	BuildspecPath string   `yaml:"buildspec"`
```

Reply:
I also thought about this. But I think it's kind of nice for them to be different and more easily distinguished, plus, does `buildspec` have to be a full URI?

Reply:
Probably not, since these are using the same functionality...maybe we should change the docs 🤔

Reply:
>  since these are using the same functionality.

Yeah, those two fields (the `buildspec` here in pre/post and the `buildspec` in `build`) are exactly the same configuration. So I feel like it's okay to have the same field name!

> maybe we should change the docs 

yeah, perhaps we should change it to something like  "The path to a buildspec file relative to the project root" 🤔